App/features/cafes/sagas.ts

```diff
@@ -135,29 +135,31 @@ function* refreshExpiredSessions() {
 }
 
 function* migrateUSW() {
-  try {
-    // Old us-west
-    const usw = '12D3KooWSsM117bNw6yu1auMfNqeu59578Bct5V4S9fWxavogrsw'
-    // New us-west
-    const repl = '12D3KooWSdGmRz5JQidqrtmiPGVHkStXpbSAMnbCcW8abq6zuiDP'
-    const cafes: Cafes = yield select((state: RootState) => state.cafes.cafes)
-    const peerIDs = Object.keys(cafes)
-  
-    if (peerIDs.indexOf(usw) > -1) {
-      // Use the existing route to deregister the usw cafe
-      yield put(actions.deregisterCafe.request({ peerId: usw }))
-      // Only replace it if there wasn't an existing secondary
-      if (peerIDs.length < 2) {
-        const cafe = cafesMap[repl]
-        if (cafe) {
-          yield put(
-            actions.registerCafe.request({ peerId: repl, token: cafe.token })
-          )
+  while (yield take([getType(TextileEventsActions.nodeOnline)])) {
+    try {
+      // Old us-west
+      const usw = '12D3KooWSsM117bNw6yu1auMfNqeu59578Bct5V4S9fWxavogrsw'
+      // New us-west
+      const repl = '12D3KooWSdGmRz5JQidqrtmiPGVHkStXpbSAMnbCcW8abq6zuiDP'
+      const cafes: Cafes = yield select((state: RootState) => state.cafes.cafes)
```

Comment:
Just because the node is online doesn't guarantee that redux will contain cafe session data. I'd just use this here:
```
const list: ICafeSessionList = yield call(Textile.cafes.sessions)
```

Reply:
sorry, confusing. to be clear. so the sessions are stored in `cafes` only if they exist but the `sessions` will store cafe details no matter what?

Reply:
oh, i see what you are saying

Reply:
maybe